AppIntegrity/Sources/AppIntegrity/FirebaseAppIntegrityService.swift

```diff
@@ -37,6 +41,7 @@ public final class FirebaseAppIntegrityService: AppIntegrityProvider {
         self.init(
```

Comment:
Not necessary for this PR but should we rename our provider factory name away from "App Attest"?

Reply:
Sure 🙂 you mean to `AppIntegrityProviderFactory`?

Reply:
Yeah or `AppCheckProviderFactory` to match the debug type from Firebase, though that may be confusing. As I say, not particularly important 🙂

Reply:
Ah, I think this is actually correct already, because it returns an `AppAttestProvider` from `Firebase`:
https://github.com/govuk-one-login/mobile-ios-one-login-app/blob/e605da49eaf2de3cbde4ce94d0e480c624d98cd2/AppIntegrity/Sources/AppIntegrity/AppAttestProviderFactory.swift#L7